pages/linux/kitex.md

```diff
@@ -0,0 +1,24 @@
+# kitex
```

Comment:
If the same command is available for more than 1 platform (Linux and macOS in this scenario), it should be placed on the `common/` directory.

Reply:
> thanks for your submission.

I know. But it is not available for  windows. So I just add page in LF & macOS.

Reply:
@baiyutang yeah, but that's the rule, if it's available for more than 1 platform, it should be placed in `common/`.